src/View/Contact/contact.html.twig

```diff
@@ -1,5 +1,6 @@
 {% extends 'layout.html.twig' %}
- 
+ 	{% block title %}Contact
```

Comment:
for better coherence keep the main title inside title block, inject him via parent twig function